packages/cli/src/commands/init/templateName.js

```diff
@@ -81,7 +81,13 @@ async function tryTemplateShorthand(templateName: string) {
       `https://registry.yarnpkg.com/${reactNativeTemplatePackage}/latest`,
     );
 
-    if (JSON.parse(response).name) {
+    if (response.status < 200 || response.status > 299) {
+      throw new Error(`Failed to load page, status code: ${response.status}`);
```

Comment:
It's a package, not page. Add some information about it

Reply:
This error message is copy pasted from previous error message that was here before. We can add some generic error handling in this case. I started a conversation in one of the previous comment. Closing this one.